### PR TITLE
[zh-cn]Update access-cluster-services.md

### PR DESCRIPTION
**Pull request by yanrongshi**
Preview：
https://github.com/kubernetes/website/pull/35414/files?short_path=3152cad#diff-3152cad8a8a955a4d885a852a4951bbdae9b6786a5088ba82dca77146792d5df

### CODE REVIEW

**Comment by Sea-n** [comment]
I think it would be better to update 3 - 10 files in a PR if you just changed `_italic_` to `**bold**`, how do you think? 🙃

**Comment by yanrongshi** [comment]
> I think it would be better to update 3 - 10 files in a PR if you just changed `_italic_` to `**bold**`, how do you think? 🙃

I have enjoyed the pleasure of merging, which may be a bad habit, but I will pay attention to it in the future

File: content/zh-cn/docs/tasks/access-application-cluster/access-cluster-services.md [hunk 1]
```diff
@@ -170,8 +170,8 @@ The supported formats for the `<service_name>` segment of the URL are:
 为了创建包含服务末端、后缀和参数的代理 URLs，你可以在服务的代理 URL 中添加：
 `http://`*`kubernetes_master_address`*`/api/v1/namespaces/`*`namespace_name`*`/services/`*`service_name[:port_name]`*`/proxy`
 
-如果还没有为你的端口指定名称，你可以不用在 URL 中指定 *port_name*。
-对于命名和未命名端口，你还可以使用端口号代替 *port_name*。
+如果还没有为你的端口指定名称，你可以不用在 URL 中指定 **port_name**。
+对于命名和未命名端口，你还可以使用端口号代替 **port_name**。
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/access-application-cluster/access-cluster-services.md:174]: 英文斜体可接受，并且和171行对应的比较好，不要改了。

File: content/zh-cn/docs/tasks/access-application-cluster/access-cluster-services.md [hunk 2]
```diff
@@ -186,9 +186,9 @@ URL 的 `<service_name>` 段支持的格式为：
...
 * To access the Elasticsearch service endpoint `_search?q=user:kimchy`, you would use:
 -->
-##### 示例
+##### 示例   {#examples}
 
-* 如要访问 Elasticsearch 服务末端 `_search?q=user:kimchy`，你可以使用：
+* 如要访问 Elasticsearch 服务末端 `_search?q=user:kimchy`，你可以使用以下地址：
 
     ```
     http://192.0.2.1/api/v1/namespaces/kube-system/services/elasticsearch-logging/proxy/_search?q=user:kimchy
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/access-application-cluster/access-cluster-services.md:194]:
````suggestion
  ```
  http://192.0.2.1/api/v1/namespaces/kube-system/services/elasticsearch-logging/proxy/_search?q=user:kimchy
````

File: content/zh-cn/docs/tasks/access-application-cluster/access-cluster-services.md [hunk 3]
```diff
@@ -197,7 +197,7 @@ URL 的 `<service_name>` 段支持的格式为：
 <!--
 * To access the Elasticsearch cluster health information `_cluster/health?pretty=true`, you would use:
 -->
-* 如要访问 Elasticsearch 集群健康信息`_cluster/health?pretty=true`，你会使用：
+* 如要访问 Elasticsearch 集群健康信息`_cluster/health?pretty=true`，你可以使用以下地址：
 
     ```
     https://192.0.2.1/api/v1/namespaces/kube-system/services/elasticsearch-logging/proxy/_cluster/health?pretty=true
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/access-application-cluster/access-cluster-services.md:203]: 同上，缩进两个空格就好

**Review by tengqm — COMMENTED** [review]
这个 PR 没有什么价值

File: content/zh-cn/docs/tasks/access-application-cluster/access-cluster-services.md (hunk 3 above)
Comment (yanrongshi) [reply content/zh-cn/docs/tasks/access-application-cluster/access-cluster-services.md:203]: Should I close it directly or modify and submit it according to your suggestion?

Comment (tengqm) [reply content/zh-cn/docs/tasks/access-application-cluster/access-cluster-services.md:203]: You can edit the file, fix it and then

```
git add content/zh-cn/docs/tasks/access-application-cluster/access-cluster-services.md
git commit --amend --no-edit
git push -f
```

**Pull request merged**